.drone.yml

```diff
@@ -189,7 +189,7 @@ steps:
 
   - name: ui-tests
     # see: docker/dockerfiles/Dockerfile
-    image: codedotorg/code-dot-org:1.10.0
+    image: codedotorg/code-dot-org:ruby-3.3.4
```

Comment:
Changing these to "real version numbers" before merge is on the pre-merge checklist